build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -164,7 +170,7 @@
       replace: ''
   when:
     - not __deploy_minio_enabled | bool
-    - __service_telemetry_observability_strategy == "use_community"
+    - not __service_telemetry_observability_strategy == "use_community"
```

Comment:
Well, this is quite confusing, but minio is getting deployed by default, when we deploy Loki and this is to **remove** the deployment. In other words, what is happening here with this PR: If observability_strategy = none, minio deployment gets removed, so it doesn't get removed. I hope I'm right, because it also took me a minute.